sys/sys/sysctl.h

```diff
@@ -1065,6 +1065,7 @@ TAILQ_HEAD(sysctl_ctx_list, sysctl_ctx_entry);
 #define	KERN_PROC_REVOKER_STATE	47	/* revoker state */
 #define	KERN_PROC_REVOKER_EPOCH	48	/* revoker epoch */
 #define	KERN_PROC_C18N		49	/* compartmentalisation statistics */
```

Comment:
Seems like it may be worth renaming this to KERN_PROC_C18N_STATS

Reply:
Yes, that would be good.